src/java/org/apache/cassandra/metrics/DroppedMessageMetrics.java

```diff
@@ -44,6 +45,56 @@ public DroppedMessageMetrics(Verb verb)
         this(new DefaultNameFactory("DroppedMessage", verb.toString()));
     }
 
+    public DroppedMessageMetrics(DefaultNameFactory factory)
+    {
+        String currentScope = factory.getScope();
+        // backward compatibility for request metrics which names have changed in 4.0 as part of CASSANDRA-15066
+        ImmutableMap<String, String> requestVerbsAlias = ImmutableMap.<String, String>builder()
+                                                         .put("BATCH_REMOVE_REQ", "BATCH_REMOVE")
+                                                         .put("BATCH_STORE_REQ", "BATCH_STORE")
+                                                         .put("COUNTER_MUTATION_REQ", "COUNTER_MUTATION")
+                                                         .put("HINT_REQ", "HINT")
+                                                         .put("MUTATION_REQ", "MUTATION")
+                                                         .put("RANGE_REQ", "RANGE_SLICE")
+                                                         .put("READ_REQ", "READ")
+                                                         .put("READ_REPAIR_REQ", "READ_REPAIR")
+                                                         .put("REQUEST_RSP", "REQUEST_RESPONSE")
+                                                         .build();
```

Comment:
I had a quick look at 3.11 and it seems to me that the metrics where containing the dropped requests and the dropped responses. If it is the case then the current approach does not fix backward compatibility because the exposed metrics will not contains the dropped responses. 
It might worth double checking what the 3.11 metrics were containing through.

Reply:
The documentation says DROPPED_MESSAGE_REQUESTS, I looked also at the MessagingService rewrite itself again:
https://issues.apache.org/jira/browse/CASSANDRA-15066?focusedCommentId=16801276&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-16801276
In the ticket Aleksey explains that "Secondly, we introduced matching response verbs for every request verb, replacing the single REQUEST_RESPONSE verb. "

I took a look at the rewrite itself again - I can see the replacement for most of them. 
I can try too double verify one more time each and everyone of them tomorrow.
Do you remember what were you looking at? Maybe I miss something for particular metric

Reply:
On the other hand I think the deprecated metric in 3.11 might have to stay until 5, as I wasn't deprecated in 3.0 but 3.11, WDYT?  PAGED_RANGE